docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+
+Next, we perform rough calibrations for ``X`` and ``SX`` gates on both qubits modeled in the
+:class:`.DynamicsBackend`, following the single-qubit calibratino tutorial for Qiskit Experiments.
+
+5.1 Configure the :class:`~qiskit.transpiler.Target` to include single qubit instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable running of the single qubit experiments, we add the following to the target:
+
+- Qubit frequency properties (needed by experiments like 
+  :class:`~qiskit_experiments.library.calibration.rough_frequency.RoughFrequencyCal`).
+- ``X`` and ``SX`` gate instructions, which the transpiler needs to check are supported by the
+  backend. 
+- Add definitions of ``RZ`` gates as phase shifts.
+
+.. jupyter-execute::
+
+    from qiskit.circuit.library import XGate, SXGate, RZGate
+    from qiskit.circuit import Parameter
+    from qiskit.providers.backend import QubitProperties
+    
+    target = backend.target
+    
+    # qubit properties
+    target.qubit_properties = [QubitProperties(frequency=v0), QubitProperties(frequency=v1)]
+    
+    # add instructions
+    target.add_instruction(XGate())
+    target.add_instruction(SXGate())
+    
+    # Add RZ instruction as phase shift for drag cal
+    phi = Parameter("phi")
+    with pulse.build() as rz0:
+        pulse.shift_phase(phi, pulse.DriveChannel(0))
+    
+    with pulse.build() as rz1:
+        pulse.shift_phase(phi, pulse.DriveChannel(1))
+    
+    target.add_instruction(
+        RZGate(phi),
+        {(0,): InstructionProperties(calibration=rz0), (1,): InstructionProperties(calibration=rz1)}
+    )
+
+5.2 Prepare :class:`~qiskit_experiments.calibration_management.calibrations.Calibrations` object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, prepare the :class:`~qiskit_experiments.calibration_management.calibrations.Calibrations`
+object.
+
+.. jupyter-execute::
+
+    import pandas as pd
+    from qiskit_experiments.calibration_management.calibrations import Calibrations
+    
+    cals = Calibrations()
+    
+    dur = Parameter("dur")
+    sigma = Parameter("σ")
+    drive = pulse.DriveChannel(Parameter("ch0"))
+    
+    # Define and add template schedules.
+    with pulse.build(name="x") as x:
+        pulse.play(pulse.Drag(dur, Parameter("amp"), sigma, Parameter("β")), drive)
+    
+    with pulse.build(name="sx") as sx:
+        pulse.play(pulse.Drag(dur, Parameter("amp"), sigma, Parameter("β")), drive)
+    
+    cals.add_schedule(x, num_qubits=1)
+    cals.add_schedule(sx, num_qubits=1)
+    
+    # add parameter guesses
+    for sched in ["x", "sx"]:
+        cals.add_parameter_value(80, "σ", schedule=sched)
+        cals.add_parameter_value(0.5, "β", schedule=sched)
+        cals.add_parameter_value(320, "dur", schedule=sched)
+        cals.add_parameter_value(0.5, "amp", schedule=sched)
```

Comment:
```suggestion
    cals = Calibrations()
    
    dur = Parameter("dur")
    sigma = Parameter("σ")
    drive = pulse.DriveChannel(Parameter("ch0"))
    
    # Define and add template schedules.
    with pulse.build(name="x") as x:
        pulse.play(pulse.Drag(dur, Parameter("amp"), sigma, Parameter("β")), drive)
    
    with pulse.build(name="sx") as sx:
        pulse.play(pulse.Drag(dur, Parameter("amp"), sigma, Parameter("β")), drive)
    
    cals.add_schedule(x, num_qubits=1)
    cals.add_schedule(sx, num_qubits=1)
    
    # add parameter guesses
    for sched in ["x", "sx"]:
        cals.add_parameter_value(80, "σ", schedule=sched)
        cals.add_parameter_value(0.5, "β", schedule=sched)
        cals.add_parameter_value(320, "dur", schedule=sched)
        cals.add_parameter_value(0.5, "amp", schedule=sched)
```
You can skip these lines by instantiating the calibration class with [a template library](https://github.com/Qiskit/qiskit-experiments/blob/0a9da6a2656515b20fcc6338207b41d1ee4274ee/qiskit_experiments/calibration_management/basis_gate_library.py#L180). Code here is correct anyways.

Reply:
Thank you - I've dropped this cell and have replaced it with one using the fixed frequency transmon template library. It changes the plots a bit due to the different default durations and sigma, but it works all the way through.